### DISCUSSION TITLE
I want to download a separate manifest, but the last one is being downloaded.

### DISCUSSION THREAD

**Discussion by BillyTill** [SteamRE/DepotDownloader#488] [Q&A] [answered]
I wrote "dotnet depotdownloader.dll -app 258550 -depot 258554 -manifest ID 6645827189265722425" this is the manifest I wanted to download, but the latest manifest is being downloaded by release date. What should I do to download exactly the manifest that I need?
Thanks in advance for the reply.

Here's what I got: 
"D:\45454545>dotnet depotdownloader.dll -app 258550 -depot 258554 -manifestid 6645827189265722425 -dir D:\bbbssssp
No username given. Using anonymous account with dedicated server subscription.
Connecting to Steam3... Done!
Logging anonymously into Steam3... Done!
Using Steam3 suggested CellID: 150
Got AppInfo for 258550
Using app branch: 'public'.
Got depot key for 258554 result: OK
Processing depot 258554
Downloading depot manifest...Got manifest request code for 258554 3335247683814853916 result: 8478604198346820337
 Done!
Manifest 3335247683814853916 (12.04.2024 20:21:43)
Downloading depot 258554"

**Comment by SlejmUr** [comment, marked as answer]
`dotnet depotdownloader.dll -app 258550 -depot 258554 -manifest 6645827189265722425`

**Reply by BillyTill** [reply]
Thanks!
